File: cpp/lab2q2.cpp
#include <iostream>
using namespace std;

int main() {
  int x;
  cout << "Type a number: ";
  cin >> x; 
  if(x>9 || x<3){
    cout<<"Invalid input";
    return 0;
  }
  for (int i=1;i<x+1;i++){
    std::string a(i, '*');
    cout<<a<<endl;
  }
  for (int i=x-1;i>0;i=i-1){
    std::string a(i, '*');
    cout<<a<<endl;
  }



  return 0;
}

File: py/lab2q2.py
a =input("Please enter a number between 3 and 9")

if(int(a)<=9 and int(a)>=3):
    for i in range(1,int(a)+1):
        print(i * "*")
    for j in range(int(a)-1,0,-1):
         print(j * "*")

else:
    print("Invalid number")
    

